spec/honeycomb/integrations/active_support_spec.rb

```diff
@@ -133,7 +133,91 @@
       end
     end
 
+    describe "custom notifications with event specific handler hook" do
+      let(:libhoney_client) { Libhoney::TestClient.new }
+      let(:configuration) do
+        Honeycomb::Configuration.new.tap do |config|
+          config.client = libhoney_client
+          config.notification_events = ["honeycomb.nonspecific_event"]
+          # rubocop:disable Metrics/LineLength
```

Comment:
If I'm picking nits, it seems like there's an awful lot of `rubocop:disable`s around this PR, especially since there are only 2 other disables in the rest of the code base:
```console
$ rg 'rubocop:disable'
spec/honeycomb/client_spec.rb
169:      # rubocop:disable Style/SymbolProc
180:        # rubocop:disable Style/SymbolProc
```
Seems like a signal that the code should be changed. But then, I'm a stickler for following rubocop - possibly more than others consider healthy. Also, I realize the lines are just gonna kind of run long with all the parameters n shit, so. 🤷

Reply:
Yeah, I am also usually a stickler for following rubocop more strictly than some. I also take it a little easier on things like LineLength or ClassLength in spec files, though. I felt like these would've been _less_ readable if split onto multiple lines somehow, especially with the wanting to keep the block params having useful names. However, happy to try to find a way to appease Rubocop without telling it to go away.

Reply:
> I also take it a little easier on things like LineLength or ClassLength in spec files

For sure.

> I felt like these would've been _less_ readable if split onto multiple lines somehow, especially with the wanting to keep the block params having useful names

Yeah, I think that's probably right. And then `ActiveSupport::Notifications` doesn't help much with its already-long name. 😂

Reply:
> ActiveSupport::Notifications doesn't help much with its already-long name.

Holy cats, it does not.

I nerfed the line length cop in a spec recently for readability, too, so I feel like we've got a precedent, at least for specs. I think I might prefer slightly that these be one-liner disables at the end of a long line (making it longer 🤦🏻 ) instead of multi-line, but ... not a blocker.

![I'll allow it!](https://user-images.githubusercontent.com/517302/123479280-b1ecd680-d5ce-11eb-995f-6e7ca8d679d9.gif)

Reply:
I honestly went back and forth on just wrapping the whole file in it or updating the rubocop.yml 😬 instead of having them scattered everywhere.